### ISSUE TITLE
Maximized windows can't be resized

### ISSUE THREAD

**Issue by nkkollaw** [elementary/gala#129]
I guess this is a feature request.

I would be great if it was possible to be able to resize maximised windows, so that if one wants all sides maximized but one (like I often do) he can maximize, and then reduce the side that he wants to reduce.

macOS allows this and it's very convenient IMHO.

**Comment by peteruithoven** [comment]
The following issue is  probably relevant for this issue: 
https://github.com/elementary/gala/issues/46

TL:DR: might be fixed with next release because of Mutter upgrade?

**Comment by nkkollaw** [comment]
@peteruithoven : it's definitely has something to do with it, although it's not the same problem.

When snapped to the side, windows MUST be able to be resized, it would make little sense otherwise. In this case, it COULD be a matter of preference whether to implement this behavior or not.

I do think it's pretty useful. It does work if you resize manually all the way.

**Comment by cassidyjames** [comment]
I don't believe this is currently possible with Mutter; in Mutter, maximization is a different state altogether than floating or resizable windows.

**Comment by nkkollaw** [comment]
The problem is that maximizing removes the window's frame, which I guess is what is used for resizing.

XFCE has an option to instruct the WM to not remove the frame.

**Comment by danrabbit** [comment]
In our case the window manager doesn’t draw window decorations, Gtk+ does that

**Comment by nkkollaw** [comment]
@danrabbit ah, got it.

I have no idea, then :-)

**Comment by lenemter** [comment]
I'll close this because the issue is old and no one showed interest in this feature last couple of years
